index.js

```diff
@@ -28,6 +28,9 @@ client.on('guildMemberAdd', (member) => {
   welcomeChannel.send(`Total Members: ${member.guild.memberCount}`);
 });
 
+// Event for when the glitter gets puked?
+client.on(/** code here */);
```

Comment:
IDK if I really understand what's going on here... I think this is probably not where we let the user hit the "puke glitter button", but maybe?